Tools/autotest/sim_vehicle.py

```diff
@@ -677,6 +677,8 @@ def start_vehicle(binary, opts, stuff, spawns=None):
         cmd.extend(["--defaults", path])
     if opts.mcast:
         cmd.extend(["--uartA mcast:"])
+    elif opts.udp:
+        cmd.extend(["--uartA udpclient:127.0.0.1:14550"])
```

Comment:
Why go anywhere near the magic 14550 port here?

```suggestion
        cmd.extend(["--uartA udpclient:127.0.0.1:5760"])
```

Reply:
good point, fixed